content/SCALE/SCALECLIReference/System/CLIKMIP.md

```diff
@@ -12,8 +12,187 @@ tags:
+| `certificate` | Enter the certificate number. System currently authenticates connection with remote KMIP Server with a TLS handshake. `certificate` and `certificate_authority` key server authentication. A valid certificate is required to verify the key server connection. WARNING: for security reasons, protect the Certificate used for key server authentication. | <code>certificate="<i>true/false</i>"</code> |
+| `certificate_authority` | Enter the CA number to use to connect with the key server. `certificate_authority` determines the certs to use to initiate the TLS handshake with `server`. A valid CA public certificate is required to authenticate the connection. WARNING: for security reasons, protect the certificate authority used for key server authentication. | <code>certificate_authority="<i>true/false</i>"</code> |
+| `port` | Enter a connection port number on the central key server. Default is `5695`. | <code>port="<i>5695</i>"</code> |
+| `server` | Enter the host name or IP address of the central key server. | <code>server="<i>hostname.com</i>"</code> |
+| `ssl_version` | Enter the option that matches the ssl configuration used by KMIP server. Options are: `PROTOCOL_TLSv1`, `PROTOCOL_TLSv1_1`, or `PROTOCOL_TLSv1_2`. | <code>ssl_version="<i>PROTOCOL_TLSv1</i>"</code> |
+| `force_clear` | Enter `true` to cancel any pending key synchronization. Cannot set to `false` if there are pending sync of existing keys unless you set `force_clear`. Set `change_server` to `true` allow users to migrate data between two KMIP servers. System first migrates keys from old KMIP server to local database and then migrate the keys from local database to new KMIP server. If it is unable to retrieve all the keys from old server, this fails. | <code>force_clear="<i>true/false</i>"</code> |
+| `change_server` | Set `change_server` to `true` to  allows users to migrate data between two KMIP servers. System first migrates keys from old KMIP server to local database and then migrate the keys from local database to new KMIP server. If it is unable to retrieve all the keys from old server, this fails. Users can bypass this by enabling `force_clear`. | <code>change_server="<i>true/false</i>"</code> |
+| `validate` | Set to `true` by default. When enabled, system tests connection to `server` making sure it can reach it. Tests the server connection and verifies the chosen certificate chain. To test, configure the `server` and `port` values, enter a `certificate` and `certificate_authority`. | <code>validate="<i>true/false</i>"</code> |
```

Comment:
The descriptions here seem to come from API docs or something and have some really awkward grammar that should be cleaned up.